addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -505,10 +506,10 @@ def _import_fattura_pa(self, tree, invoice):
             if document_date:
                 invoice.invoice_date = document_date
             else:
-                message_to_log.append("%s<br/>%s" % (
+                message_to_log.append(Markup("%s<br/>%s" % (
```

Comment:
Markup can now be around the `Markup("%s<br/>%s") % (...)` as the result of `_compose_info_message` should be markup thanks to the change on it.

Reply:
@prra-odoo should have specified, that comment applied to all similar code in the commit

Reply:
@reth-odoo nah,  my bad 😅 I didn't see other similar changes, anyways I have made the changes. Thanks :)